renovate.json

```diff
@@ -0,0 +1,37 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": ["config:base"],
+  "includePaths": ["packages/create-plugin/templates/**"],
+  "separateMajorMinor": false,
+  "enabledManagers": ["regex"],
+  "customManagers": [
+    {
+      "customType": "regex",
+      "fileMatch": ["_package.json"],
+      "matchStrings": ["\"(?<depName>@grafana/.*)\"[^\"]*:\\s*\"\\^(?<currentValue>.*)\""],
+      "datasourceTemplate": "npm"
+    },
+    {
+      "customType": "regex",
+      "fileMatch": ["plugin.json"],
+      "matchStrings": ["\"grafanaDependency\":\\s*\"\\>=(?<currentValue>.*)\""],
+      "depNameTemplate": "grafana/grafana-enterprise",
+      "datasourceTemplate": "docker"
+    },
+    {
+      "customType": "regex",
+      "fileMatch": ["(^|/)(?:docker-)?compose[^/]*\\.ya?ml$"],
+      "matchStrings": ["grafana_version:\\s\\$\\{GRAFANA_VERSION:-(?<currentValue>.*)\\}"],
+      "depNameTemplate": "grafana/grafana-enterprise",
+      "datasourceTemplate": "docker"
+    }
+  ],
+  "packageRules": [
+    {
+      "matchPackagePatterns": ["*"],
+      "matchUpdateTypes": ["minor", "patch", "major"],
```

Comment:
We patch old versions of Grafana. Do we know if they will trigger a renovate PR or does it know to only "look ahead"?

Reply:
Hmm can't find any docs backing this now, but I'm pretty sure that wont't be the case. Going from for example 11.0.0 to 10.4.9 would be a downgrade and that should not be allowed (at least not by default).